src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/SmeltersTool.java

```diff
@@ -0,0 +1,68 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.tools;
+
+import io.github.thebusybiscuit.slimefun4.api.items.ItemGroup;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItemStack;
+import io.github.thebusybiscuit.slimefun4.api.recipes.RecipeType;
+import io.github.thebusybiscuit.slimefun4.core.attributes.DamageableItem;
+import io.github.thebusybiscuit.slimefun4.core.handlers.ToolUseHandler;
+import io.github.thebusybiscuit.slimefun4.implementation.Slimefun;
+import io.github.thebusybiscuit.slimefun4.implementation.items.SimpleSlimefunItem;
+import io.github.thebusybiscuit.slimefun4.utils.tags.SlimefunTag;
+import me.mrCookieSlime.Slimefun.api.BlockStorage;
+import org.bukkit.Effect;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.inventory.ItemStack;
+
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.util.Collection;
+import java.util.Optional;
+
+public class SmeltersTool extends SimpleSlimefunItem<ToolUseHandler> implements DamageableItem {
+
+    @ParametersAreNonnullByDefault
+    public SmeltersTool(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe) {
+        super(itemGroup, item, recipeType, recipe);
+    }
+
+    @Override
+    public @Nonnull ToolUseHandler getItemHandler() {
+        return (e, tool, fortune, drops) -> {
+            Block block = e.getBlock();
+            Material blockType = block.getType();
+
+            if (block.isPreferredTool(tool) && SlimefunTag.SMELTERS_BLOCKS.isTagged(blockType) && BlockStorage.hasBlockInfo(block)) {
```

Comment:
You're currently *only* smelting blocks that ARE Slimefun blocks

Reply:
oof i forgot ! mb